apis/v1alpha3/types.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // A ProviderSpec defines the desired state of a Provider.
 type ProviderSpec struct {
-	runtimev1alpha1.ProviderSpec `json:",inline'"`
+	runtimev1alpha1.ProviderSpec `json:",inline"`
```

Comment:
This appears to have been included in previous release, but did not affect functionality from what I can tell.